.github/workflows/community.yml

```diff
@@ -0,0 +1,30 @@
+on:
+  issues:
+    types: [ opened ]
+  issue_comment:
+    types: [ created ]
+  pull_request:
+    types: [ opened ]
+  pull_request_review:
+    types: [ submitted ]
+  pull_request_review_comment:
+    types: [ created ]
+
+jobs:
+  welcome:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/first-interaction@v1
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          issue-message: "<h1>Looks good having you contribute to this project!</h1> Feel free to raise an <strong>Issue</strong>! Welcome to the community :nerd_face: If any questions please ask in [Discord](https://discord.com/invite/jZQs6Wu)"
+          pr-message: "<h1>Looks good having you contribute to this project!</h1> Feel free to create a <strong>Pull Request</strong>! Welcome to the community :nerd_face: If any questions please ask in [Discord](https://discord.com/invite/jZQs6Wu)"
+
+  stats:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: EddieJaoudeCommunity/gh-action-community@main
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I don't think statistics needs access to the GitHub token.

It's not in the example here:
https://github.com/EddieJaoudeCommunity/gh-action-community

Reply:
I think you are right @AllanRegush, it only requires the firebase key [here](https://github.com/EddieJaoudeCommunity/gh-action-community/blob/ce8472da914534839adae6628f75b5b8d2aa1451/.github/workflows/community.yml#L31)

Reply:
Good spot, yep this is not needed on the `statistics` Action 👍